### PR TITLE
Support task to Resolve names/IPs on Unix/Linux

### PR DESCRIPTION
**Pull request by shadeon**
This PR adds a unix task that allows you to resolve a list of IP Addresses and/or names into complete lookup records from the target server's perspective.

## Changes

- Added a probe module that supports querying DNS on the local machine with a list of targets, as supplied via a parameter.
- Added a task that exposes the above probe, where users can specify addresses via the override-able parameter.

## Issues Addressed
#10 
## Breaking Changes
None.

### CODE REVIEW

File: ManagementPacks/Community.DataOnDemand.Unix/Scripts/ResolveAddress.pl [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    if (_is_ipv4($query)) {            
+        my $hostname = gethostbyaddr(inet_aton($query), AF_INET);
+        if ($hostname){
+            print "$query,$hostname$lineEnd";
+        }    
+    }
+    else {        
+        my $ipaddr = gethostbyname($query);
+        if ($ipaddr) {
+            $ipaddr = inet_ntoa(inet_aton($query));
```
Comment (clarkd) [inline ManagementPacks/Community.DataOnDemand.Unix/Scripts/ResolveAddress.pl]: What's the benefit if doing inet_ntoa only to do the opposite again?

File: ManagementPacks/Community.DataOnDemand.Unix/Modules/Probe.ResolveAddress.mpx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  <LanguagePacks>
+    <LanguagePack ID="ENU" IsDefault="true">
+      <DisplayStrings>
+        <DisplayString ElementID="Community.DataOnDemand.Unix.Probe.ResolveAddress">
+          <Name>Data On Demand Unix resolve address probe action</Name>
+          <Description>Resolves the listed IP addresses or names on the target computer.</Description>
+        </DisplayString>
+        <DisplayString ElementID="Community.DataOnDemand.Unix.Probe.ResolveAddress" SubElementID="Addresses">
+          <Name>Addresses</Name>
+          <Description>IP addresses or name to resolve, as a comma seperated list.</Description>
```
Comment (clarkd) [inline ManagementPacks/Community.DataOnDemand.Unix/Modules/Probe.ResolveAddress.mpx]: 'names'

**Review by clarkd — CHANGES_REQUESTED** [review]

File: ManagementPacks/Community.DataOnDemand.Unix/Scripts/ResolveAddress.pl [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+# Although Socket could be used instead of regex and manual validation, the behaviour and availability is not platform consistent.
+sub _is_ipv4 {
+    shift if ref $_[0];
+    my $value = shift;
+
+    return undef unless defined($value);
+
+    my (@octets) = $value =~ /^(\d{1,3})\.(\d{1,3})\.(\d{1,3})\.(\d{1,3})$/;
```
Comment (DavidR91) [inline ManagementPacks/Community.DataOnDemand.Unix/Scripts/ResolveAddress.pl:46]: I feel like this line, along with 48 and 49 could do with a small comment just explaining roughly what their effects  [on each octet] are  (esp. given nobody really knows Perl)

**Review by DavidR91 — APPROVED** [review]
Only suggestion would be commentary in the Perl file (see comment)

**Review by clarkd — APPROVED** [review]

**Pull request merged**